bedrock/mozorg/templates/mozorg/home/home-b.html

```diff
@@ -262,4 +262,8 @@ <h1 class="logo"><a href="{{ url('mozorg.home') }}" data-link-type="footer" data
 
 {% block js %}
   {% javascript 'home-b' %}
+
+  {% if LANG == 'en-US' %}
```

Comment:
Hm, there is a locale restriction, but it's in Optimizely (not in the view). I suppose this could be removed for now.